### PR TITLE
feat: get balancer from config

### PR DESCRIPTION
**Pull request by emretanriverdi**
I've added it as a string in contrast to general usage as int, but I've seen int usage is also default in segmentio configuration yml; but this parameter is normally not taken in config; that's why I added it as string to make it clearer.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -32,6 +43,13 @@ type Kafka struct {
 	AllowAutoTopicCreation      bool              `yaml:"allowAutoTopicCreation"`
 }
 
+func (k *Kafka) GetBalancer() kafka.Balancer {
+	if balancer, ok := balancerMap[k.Balancer]; ok {
+		return balancer
+	}
+	panic("Invalid kafka balancer method")
```
Comment (mhmtszr) [inline config/config.go:50]: We can also write k.Balancer here to be able to see.

File: config/config.go [hunk 2]
```diff
@@ -4,11 +4,21 @@ import (
...
 	"time"
 
-	"github.com/Trendyol/go-dcp/helpers"
-
 	"github.com/Trendyol/go-dcp/config"
+	"github.com/Trendyol/go-dcp/helpers"
+	"github.com/segmentio/kafka-go"
 )
 
+var balancerMap = map[string]kafka.Balancer{
```
Comment (mhmtszr) [inline config/config.go:12]: Can we write these balancers strings to README.md as well?

File: config/config.go [hunk 3]
```diff
@@ -4,11 +4,21 @@ import (
...
+	"":                &kafka.Hash{},
+	"Hash":            &kafka.Hash{},
```
Comment (mhmtszr) [inline config/config.go:14]: Instead of storing reference of every hash types we may generate it by switch case what do you think?

Comment (emretanriverdi) [reply config/config.go:14]: Good idea

File: config/config.go (hunk 2 above)
Comment (emretanriverdi) [reply config/config.go:12]: Yup

File: config/config.go (hunk 1 above)
Comment (emretanriverdi) [reply config/config.go:50]: Done

**Review by mhmtszr — APPROVED** [review]
LGTM!

**Pull request merged**